### PR TITLE
Nervos CKB Development Update #37

### PR DESCRIPTION
**Pull request by doitian**

### CODE REVIEW

File: 2020/2020-06-16_Nervos-CKB-Development-Update-37.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+[[@doitian SUGGEST CHANGE HERE]]
+
+## Changes in RFCs
+
+[[@doitian SUGGEST CHANGE HERE]]
+
+## Changes in CKB
+
+[[@doitian SUGGEST CHANGE HERE]]
```
Comment (doitian) [inline 2020/2020-06-16_Nervos-CKB-Development-Update-37.md]:
```suggestion
We have released [v0.32.1](https://github.com/nervosnetwork/ckb/releases/tag/v0.32.1) and [v0.32.2](https://github.com/nervosnetwork/ckb/releases/tag/v0.32.2) with bug fixings. The  v0.33.0 code has been frozen.

We continue working on RPC documentations, chain freezer, light client and test.
```

File: 2020/2020-06-16_Nervos-CKB-Development-Update-37.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Nervos CKB Development Update #37
+
+Covering Jun 1 to Jun 14, 2020
+
+By [Cipher Wang](https://github.com/CipherWang), [Xuejie Xiao](https://github.com/xxuejie), [Ian Yang](https://github.com/doitian).
+
+## TL;DR
+
+[[@doitian SUGGEST CHANGE HERE]]
```
Comment (doitian) [inline 2020/2020-06-16_Nervos-CKB-Development-Update-37.md]:
```suggestion
* We have released [v0.32.1](https://github.com/nervosnetwork/ckb/releases/tag/v0.32.1) and [v0.32.2](https://github.com/nervosnetwork/ckb/releases/tag/v0.32.2) with bug fixings
```

File: 2020/2020-06-16_Nervos-CKB-Development-Update-37.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Changes in RFCs
+
+[[@doitian SUGGEST CHANGE HERE]]
+
```
Comment (doitian) [inline 2020/2020-06-16_Nervos-CKB-Development-Update-37.md]:
```suggestion
```

**Comment by doitian** [comment]
ping @xxuejie @CipherWang

File: 2020/2020-06-16_Nervos-CKB-Development-Update-37.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+## Changes in CKB
+
+[[@doitian SUGGEST CHANGE HERE]]
+
+## Changes in DT
+
+[[@xxuejie SUGGEST CHANGE HERE]]
```
Comment (xxuejie) [inline 2020/2020-06-16_Nervos-CKB-Development-Update-37.md]:
```suggestion
[[@xxuejie SUGGEST CHANGE HERE]]

We are busy preparing launches for several projects by the end of month! Please stay tuned.
```

**Pull request merged**